getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Events in Axiom"
+description: "This page explains the fundamentals of event data in Axiom."
+sidebarTitle: Events
+---
+
+In Axiom, event data is the atomic unit of activity, encompassing logs, traces, and even product interactions. Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, from log lines to complex traces, they usually contain the following:
```

Comment:
In the past, I have said that event data is the record of every interaction between human, sensor, and machine. i can’t remember where that originated from, either cfrln or a deep dive on the history of event data ~Q1/Q2, but the reason I gravitated towards it is because it zooms us out from any use case first, to create an appreciation for the total scope and opportunity before honing in. 

By putting logs and traces as the first and second example, it feels to me that we have already coupled Axiom so tightly to engineering use cases, and then we try and walk it back with the third example, product interactions. I really want the reader to feel that timestamped event data is _the_ record of absolutely every digital interactional - the digital breadcrumb - and _then_ (and only then) for us to contextualize that with domain-specific examples. That could work better as two sentences.